docs/source/audio_load.mdx

```diff
@@ -55,3 +55,126 @@ If you only want to load the underlying path to the audio dataset without decodi
  'transcription': 'I would like to set up a joint account with my partner'}
 ```
 
+## AudioFolder
+
+You can also load a dataset with an `AudioFolder` dataset builder. It does not require writing a custom dataloader, making it useful for quickly loading audio data. Your dataset structure might look like:
```

Comment:
I think you can remove this part, since it's already presented later in "AudioFolder with labels"